baselines/FedStar/FedStar/server.py

```diff
@@ -0,0 +1,219 @@
+##########################################
+import os
+import sys
+
+os.environ["TF_CPP_MIN_LOG_LEVEL"] = "3"
+os.environ["GRPC_VERBOSITY"] = "ERROR"
+GRPC_MAX_MESSAGE_LENGTH: int = 536_870_912
+##########################################
+import argparse
+import pathlib
+import uuid
+import gc
+import time
+import flwr
+import logging
+import tensorflow as tf
+from model import Network
+from data import DataBuilder
```

Comment:
Once you have done the renaming, when you build your environment with `poetry`, a python package will be built for `fedstar`. In this way, you can easily import your modules from anywhere in the code or elsewhere. 

This means that the lines above (and all others in your code that import from the files you wrote) should be imported as:
```python
from fedstar.model import Network
from fedstar.data import DataBuilder
```

Reply:
thanks for the suggestion, have updated the imports accordingly.